### PR TITLE
Js intermediate stuff

### PR DESCRIPTION
**Pull request by mikoscz**
Add two js exercises

### CODE REVIEW

File: exercises/js_intermediate_stuff/konami_code/the_konami_code.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Source of data
+
+As a smart hacker, you've found a safe place to communicate - names of GitHub issues. Every day your teammates are posting
+issues on a different repository and the name of every one is another secret message. For today you and your team have choosed
+elixir language repository. To get the data just fetch JSON form this URL: https://api.github.com/repos/elixir-lang/elixir/issues
+
+## Prerequisites
+1) 'ESC' key cancels currently written part of the Konami Code
+2) Max timestamp beetween each next pressed key is 5 seconds(after that time Konami Code should be reseted)
```
Comment (socjopata) [inline exercises/js_intermediate_stuff/konami_code/the_konami_code.md]: reseted -> reset

File: exercises/js_intermediate_stuff/rainbow_table/rainbow_table.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+You and your group of hackers, get the secret database of the Linkedin company. One of the records from the DB caught your
+attention especially:
+
+Name | Email | Password
+--- | --- | ---
+Dariusz Wylon | d.wylon@selleo.com | c8e095e2a26f8540afabb36dcdaee3b1
+
+It looks like you have credentials but, ehmm.. the password is encrypted. The good news is that one of the team members get
+information about which encryption algorithm was used - it was MD5. This is a great news but decrypting this via brute forcing this will take years.
```
Comment (socjopata) [inline exercises/js_intermediate_stuff/rainbow_table/rainbow_table.md]: Consider rewrite -> "but decrypting this via brute forcing this"

**Review by socjopata — APPROVED** [review]
These would be very nice additions, thanks! :)

File: exercises/js_intermediate_stuff/konami_code/the_konami_code.md (hunk 1 above)
Comment (mikoscz) [reply exercises/js_intermediate_stuff/konami_code/the_konami_code.md]: Nice catch ;)

File: exercises/js_intermediate_stuff/rainbow_table/rainbow_table.md (hunk 2 above)
Comment (mikoscz) [reply exercises/js_intermediate_stuff/rainbow_table/rainbow_table.md]: Done

**Pull request merged**